src/components/predictive-layer/predictive-layer.vue

```diff
@@ -1,201 +1,184 @@
 <template>
-  <Empathize :animation="empathizeAnimation" class="x-list">
-    <BaseKeyboardNavigation
-      class="x-row x-row--gap-06 x-row--align-start"
-      :navigationHijacker="navigationHijacker"
-    >
-      <IdentifierResults
-        v-if="showIdentifierResults"
-        v-slot="{ identifierResult }"
-        :maxItemsToRender="5"
-        :animation="suggestionsAnimation"
-        class="x-row__item x-row__item--span-12 x-list x-list-horizontal x-padding--05"
-        :class="$x.device === 'mobile' ? 'x-list--gap-03 x-padding--top-00' : 'x-list--gap-02'"
-      >
-        <BaseResultLink
-          v-slot="{ result }"
-          :result="identifierResult"
-          class="x-suggestion-lg x-suggestion desktop:x-suggestion-md"
+  <Empathize :animation="empathizeAnimation" class="x-bg-neutral-0 desktop:x-py-16 desktop:x-pl-16">
+    <component :is="$x.device === 'desktop' ? 'div' : 'BaseScroll'" class="x-h-full">
```

Comment:
With this change, the empathize will paint a white block:

![image](https://user-images.githubusercontent.com/6597815/222388154-b238d8d3-099c-40b1-b3fc-d5ad7b7af482.png)

Reply:
true, we have to create 2 different components for predictive layer, one for Mobile and another for Desktop

Reply:
The white block is displayed because of the padding that is added no matter if the Empathize has content or not, so we could add it conditionally. 

```
    :class="{ 'desktop:x-py-16 desktop:x-pl-16': showEmpathize }"
```

Anyways, it is only happening when there are neither Query Suggestions nor Next Queries, otherwise, the Next Queries will be displayed:

![image](https://user-images.githubusercontent.com/77450928/222402729-34c1ae8b-dad6-43d1-971b-6d39bf1fc916.png)

I was thinking that, in the scenario of no query suggestions, maybe we should display Top Clicked and Popular Searches instead of not showing anything at all in the predictive. Because not showing at all is causing this empty screen (until enter key is pressed):

![image](https://user-images.githubusercontent.com/77450928/222403487-6ba1d7be-3741-432b-b82f-7dead41d0079.png)

Reply:
Yes, the space is because of padding. But avoiding padding is easily "breakable" on styles change during setup.

The empathize controls if something is rendered inside or not, and if does have not anything inside then it is not rendered.

The `<component>` there making this problem is because we want different components and behaviour on mobile and desktop.

That's the source of the problem we have to solve.

Changing the content to avoid empty case, is not also a solution as also is easily "breakable" on a setup.

Reply:
You're right @tajespasarela, avoiding padding is easily "breakable". But I don't get why we need to create two different components for the predictive layer. We just need to render the `<component>` conditionally so the Empathize is not rendered if has no content, right?

Reply:
Is something we have to do. It would be better to do on another task. so if you want to solve this in another way, go ahead, but this is something we have to do eventually.